src/api_server/src/parsed_request.rs

```diff
@@ -315,7 +315,7 @@ pub(crate) mod tests {
     use vmm::rpc_interface::VmmActionError;
     use vmm::vmm_config::balloon::{BalloonDeviceConfig, BalloonStats};
     use vmm::vmm_config::instance_info::InstanceInfo;
-    use vmm::vmm_config::machine_config::VmConfig;
+    use vmm::vmm_config::machine_config::MachineConfig;
```

Comment:
`MachineConfig` is not defined until this commit c281c0641b42222d4872cc24d72b02d0e9222071, introduced in commit 475654ba992078714c03ffda92712f5e1a6babfc. Each commit should be able to be compiled.

> - Renamed VmConfig into MachineConfig

It looks to me that not just renaming `VmConfig` into `MachineConfig`, but integrating 2 data structures `VmConfig` and `VmUpdateConfig` into `MachineConfig`. Is this understanding correct?

And it would be great to make commit messages have not only what changes are included but also why those changes are introduced.

Reply:
Redid commits